### DISCUSSION TITLE
Astro embed not working

### DISCUSSION THREAD

**Discussion by soranauts** [arthelokyo/astrowind#323] [Q&A] [answered]
Any reason why [Astro Embed](https://github.com/delucis/astro-embed) wouldn't be working or am I doing something wrong? All the code is correct. It was very minimal to get started with this integration. 

Twitter links embed but not video for .mdx files. It works in .astro files though.

It's odd because the HTML shows the video is embedded.

Here's some open issues on this.

https://github.com/withastro/astro/issues/7761

https://github.com/delucis/astro-embed/issues/57

Maybe it's not an issue with the theme. Would you happen to know of a workaround?  Thanks.

**Comment by prototypa** [comment]
Hello @soranauts. We are moving this to Discussion because it is not an issue with **AstroWind**.

**Comment by colemilne54** [comment]
Running into the same issue, have yet to find a solution.

**Comment by prototypa** [comment, marked as answer]
Hi @soranauts and @colemilne54 

You were right, it was a technical detail in AstroWind. It is fixed:
https://github.com/onwidget/astrowind/issues/420

Now `astro-embed` is installed by default and is used in some pages:
- https://astrowind.vercel.app/homes/startup
- https://astrowind.vercel.app/markdown-elements-demo-post#astro-embed
 
Any problem let me know.
Thanks
